<!DOCTYPE html>
<html><head><title>exceptionMessage</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">3 references to exceptionMessage</div><div class="rA">Azure.Security.ConfidentialLedger (3)</div><div class="rG" id="Azure.Security.ConfidentialLedger"><div class="rF"><div class="rN">PostLedgerEntryOperation.cs (3)</div>
<a href="../PostLedgerEntryOperation.cs.html#69"><b>69</b>? await _client.clientDiagnostics.CreateRequestFailedExceptionAsync(statusResponse, <i>exceptionMessage</i>).ConfigureAwait(false)</a>
<a href="../PostLedgerEntryOperation.cs.html#70"><b>70</b>: _client.clientDiagnostics.CreateRequestFailedException(statusResponse, <i>exceptionMessage</i>);</a>
<a href="../PostLedgerEntryOperation.cs.html#71"><b>71</b>return OperationState.Failure(GetRawResponse(), new RequestFailedException(<i>exceptionMessage</i>, ex));</a>
</div>
</div>
</body></html>